### ISSUE TITLE
Metahandler

### ISSUE THREAD

**Issue by JUL1EN094** [Eldorados/script.module.metahandler#3]
Hi Eldorados,

I'll need to bump official version with lastest commits from Kubakul. Are u still supporting the script ? Any way to bump the version number and push it to official repo ?

Thank you

**Comment by Eldorados** [comment]
Yes I'm still supporting, I have some changes that I just noticed I haven't committed yet

Which commits from Kubakul are you referring?

**Comment by JUL1EN094** [comment]
All commit since v2.0.9 : 

https://github.com/Eldorados/script.module.metahandler/commit/ccd3b6107923e83dca76ea3f8333343f48782580

https://github.com/Eldorados/script.module.metahandler/commit/42abc33b882e0dc59ab3f753ff0eec95454f8c00

https://github.com/Eldorados/script.module.metahandler/commit/5e926185f69cff978771ad75f94b0443df009954


https://github.com/Eldorados/script.module.metahandler/commit/a5bec0b110e7dcda57b20fb3debb616649bbf71e

Thank you
